contentcuration/contentcuration/view/registration_views.py

```diff
@@ -45,7 +46,7 @@ def send_invitation_email(request):
             ctx_dict = {    'sender' : request.user,
                             'site' : get_current_site(request),
                             'user' : recipient,
-                            'share_mode' : share_mode,
+                            'share_mode' : _(share_mode),
```

Comment:
Only need either the `_()` here or the `trans` tag above, not both. But probably won't do any harm!